examples/README.md

```diff
@@ -0,0 +1,60 @@
+# examples
+
+This directory contains example code for both installing Contour and its associated Envoy Pods under various conditions, and also for setting up various extensions and other pieces of software that may help with running Contour.
+
+## Directories
```

Comment:
I think this section heading is redundant, this was covered in the sentence above. delete lines 5-8